docs/translating/polish.md

```diff
@@ -2,6 +2,8 @@
 
 Use semi-formal Polish for translation, some specifics:
 
+*we: my
```

Comment:
There is a space missing after the '*'.

Reply:
Can you some info explaining when to use or not to use "my" in Polish translations?